spec/mailers/weby_mailer_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.